tests/Functional/Controllers/Backend/NotificationTest.php

```diff
@@ -50,20 +59,28 @@ public function setUp(): void
     protected function tearDown(): void
     {
         parent::tearDown();
-        $sql = 'DELETE FROM s_articles_notification WHERE id IN (1111111111, 1111111112, 1111111113)';
+        $sql = 'DELETE FROM s_articles_notification WHERE id IN (1111111111, 1111111112, 1111111113, 1111111114)';
         Shopware()->Db()->query($sql);
     }
 
     /**
      * test getList controller action
      */
-    public function testGetArticleList()
+    public function testGetArticleListFirstPage()
     {
-        $this->dispatch('backend/Notification/getArticleList');
+        $parameter = [
+            'limit' => self::NOTIFICATION_LIST_PAGINATION_LIMIT,
+            'start' => self::NOTIFICATION_LIST_PAGINATION_START_1,
+        ];
+        $getParameter = http_build_query($parameter);
+
+        $this->dispatch('backend/Notification/getArticleList?' . $getParameter);
         static::assertTrue($this->View()->success);
         $returnData = $this->View()->data;
         static::assertNotEmpty($returnData);
-        static::assertCount(2, $returnData);
+
+        static::assertCount(self::NOTIFICATION_LIST_PAGINATION_ITEMS_COUNT_1, $returnData);
+        static::assertEquals(self::NOTIFICATION_LIST_TOTAL_RESULTS, $this->View()->totalCount);
```

Comment:
```suggestion
        static::assertEquals(self::NOTIFICATION_LIST_TOTAL_RESULTS, $this->View()->getAssign('totalCount'));
```

do not use magic property access, but this should also be comment on by PHPStan if the pipeline would run :grimacing: